packages/api/__tests__/headers.test.ts

```diff
@@ -0,0 +1,224 @@
+import { Api } from '../src';
+import { IApiTestsListData } from './types';
+
+const { TEST_WORKSPACE_ID } = process.env;
+
+describe('headers', () => {
+  let originalFetch;
+
+  beforeAll(() => {
+    originalFetch = fetch;
+    // @ts-ignore
+    fetch = jest.fn((...args) => originalFetch(...args));
+  });
+
+  afterEach(() => {
+    // @ts-ignore
+    fetch.mockClear();
+  });
+
+  afterAll(() => {
+    // @ts-ignore
+    fetch = originalFetch;
+  });
+
+  it('passes headers from constructor. Object', () => {
+    const api = new Api({
+      workspaceId: TEST_WORKSPACE_ID,
+      headers: {
+        auth: 'Some token',
```

Comment:
Let's name the key `authentication` so that it's consistent with the bearer token key name.

Reply:
It's just a test. Any key will suite the needs.

Reply:
Got it, so the expectation here is that the user still passes the proper athentication header? If so, that's fine.

Reply:
Got it, so the expectation here is that the user still passes the proper athentication header? If so, that's fine.

Reply:
Got it, so the expectation here is that the user still passes the proper athentication header? If so, that's fine.

Reply:
I just check ability to pass any headers from request to fetch.